app/src/main/java/com/hedvig/app/feature/offer/OfferViewModel.kt

```diff
@@ -43,16 +47,15 @@ class OfferViewModelImpl(
     }
 
     fun load() {
-        disposables += offerRepository
```

Comment:
Make sure to remove `CompositeDisposable`s and `onCleared` from `ViewModel`s when they are no longer necessary 🙂